### DISCUSSION TITLE
Icarus 有添加 pjax 的计划吗

### DISCUSSION THREAD

**Discussion by FreeDivers** [ppoffice/hexo-theme-icarus#1118] [General]
每次进入网页的其中一个页面都要整个网页全部加载，观感不是很好
有看到 [#992](https://github.com/ppoffice/hexo-theme-icarus/issues/992) 的建议，不过现在还没有见加上，想问一下有没有这方面的计划。

**Comment by PJ-568** [comment]
已并入主线：[#1287](https://github.com/ppoffice/hexo-theme-icarus/pull/1287) 。
